From 3804f5bb4acfa7e58d0a0890225d8584422f3629 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 15 Jun 2018 07:01:26 +0200
Subject: [PATCH 072/144] common: print \n in initr_scsi()

Typically init_scsi() does not output anything. So initr_scsi() should
provide a \n or we may see borked output like

	SCSI:  Net:   No ethernet found.

as observed with sandbox_defconfig.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 common/board_r.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/board_r.c b/common/board_r.c
index 6949d4af0e..e9f827a741 100644
--- a/common/board_r.c
+++ b/common/board_r.c
@@ -553,6 +553,7 @@ static int initr_scsi(void)
 {
 	puts("SCSI:  ");
 	scsi_init();
+	puts("\n");
 
 	return 0;
 }
-- 
2.18.0

